src/main/java/Duke.java

```diff
@@ -1,10 +1,193 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void printEntireList(Task[] list) {
+        int i = 0;
+        for(Task item: list) {
+            if(item != null) {
+                System.out.println((i+1) + ". "+ item.toString());
+                i++;
+            }
+        }
+    }
+
+    public static void printReplyForAddCommand(int counterList, String DASH_LINE, String s) {
```

Comment:
I think that the naming for String s can be changed to a more descriptive name such as taskToAdd, what do you think?

Reply:
Yep, agree with you, thank you for the suggestion!